WebBackend/app.py

```diff
@@ -0,0 +1,76 @@
+import json
+
+from cerberus import Validator
+import configparser
+from flask import Flask, request
+from flask_limiter import Limiter
+from flask_limiter.util import get_remote_address
+import joblib
+import requests
+
+config = configparser.RawConfigParser()
+config.read('settings.cfg')
+settings = dict(config.items('sprinters-config'))
+API_KEY = settings['api_key']
+model = joblib.load(settings['model_path'])
+encoder = joblib.load(settings['encoder_path'])
+
+fields = ['offer_type', 'floor', 'area', 'rooms', 'offer_type_of_building', 'market', 'longitude', 'latitude']
+
+schema = {
+    'offer_type': {'type': 'string', 'required': True},
+    'floor': {'type': 'float', 'min': -1.00, 'required': True},
+    'area': {'type': 'float', 'min': 10.00, 'required': True},
+    'rooms': {'type': 'integer', 'min': 1, 'required': True},
+    'offer_type_of_building': {'type': 'string', 'required': True},
+    'market': {'type': 'string', 'required': True},
+    'longitude': {'type': 'float', 'required': True},
+    'latitude': {'type': 'float', 'required': True}
+}
+validator = Validator(schema)
+
+app = Flask(__name__)
+
+limiter = Limiter(
+    app,
+    key_func=get_remote_address,
+    default_limits=['1/5seconds']
+)
+
+def _get_location_data(latitude, longitude):
+    url = 'https://geocodeapi.p.rapidapi.com/GetNearestCities'
+    location_fields = ['Population', 'CountryId']
```

Comment:
I think `City` (`city_name`) also could be included, since we already getting this in response.

Reply:
Added field city_name